File: library/src/main/java/com/yy/codex/uikit/NSShadow.java
package com.yy.codex.uikit;

import android.graphics.Color;

/**
 * Created by cuiminghui on 2017/1/9.
 */

public class NSShadow {

    public CGSize shadowOffset = new CGSize(1, 1);
    public double shadowBlurRadius = 1;
    public int    shadowColor = Color.BLACK;

    public NSShadow() {}

    public NSShadow(CGSize shadowOffset, double shadowBlurRadius, int shadowColor) {
        this.shadowOffset = shadowOffset;
        this.shadowBlurRadius = shadowBlurRadius;
        this.shadowColor = shadowColor;
    }

}

File: library/src/main/java/com/yy/codex/uikit/NSShadow.kt
package com.yy.codex.uikit

import android.graphics.Color

/**
 * Created by cuiminghui on 2017/1/9.
 */

class NSShadow {

    var shadowOffset = CGSize(1.0, 1.0)
    var shadowBlurRadius = 1.0
    var shadowColor = Color.BLACK

    constructor() {}

    constructor(shadowOffset: CGSize, shadowBlurRadius: Double, shadowColor: Int) {
        this.shadowOffset = shadowOffset
        this.shadowBlurRadius = shadowBlurRadius
        this.shadowColor = shadowColor
    }

}
